democracylab_environment_variables.sh

```diff
@@ -108,6 +108,6 @@ export DISALLOW_CRAWLING=True
 life, focusing on creating new customers, reinvigorating existing customers and increasing revenue.\n\nDwayne is a Certified Customer Experience Professional (CCXP), with proven ability to create award-winning customer experiences for recognizable brands and startups alike. Some examples include Microsoft, Nike, Cisco, Healthcare Corporation of America, Boeing, and National Institutes of Health, among others."},{"first_name":"Daria","last_name":"Loi","title":["", "Senior Director, Head of Product Design at Mozilla"],"user_thumbnail":"https://d1agxr2dqkgkuy.cloudfront.net/img/Daria%20Loi.jpg","bio_text":"Daria loves mixing design strategy with user experience research & innovation to enrich people’s life & humanize technology.\n\nShe is a passionate & accomplished creative & technical leader, versed applying UX, tech & design acumen to diverse contexts & experienced deploying innovation to dream, design & develop novel products & services.\n\nShe has broad strategic & operational expertise in planning, project, & innovation management & in navigating highly ambiguous contexts to deliver results.\n\nIn 2018 Daria was recognized as one of Italy’s 50 most inspiring women in tech&received Intel’s Global Diversity&Inclusion Award."},{"first_name":"Elizabeth","last_name":"Scallon","title":["", "Head of WeWork Labs, Northern California (SF) & Northwest"],"user_thumbnail":"https://d1agxr2dqkgkuy.cloudfront.net/img/Elizabeth%20Scallon.jpg","bio_text":"Elizabeth is an award winning leader in the Pacific Northwest startup ecosystem, and Head of WeWork Labs Northern California & Northwest. She is a scientist, mentor, and 15 year leader in scientific and technological innovation.\n\nKnown for a keen ability to locate disruptive and innovative technology, previous to WeWork Elizabeth led the team at CoMotion Labs at the University of Washington coaching and growing the robust bench of NW tech and biotech startups.\n\nAn out lesbian from before the L Word pilot episode, Elizabeth has a passion for spotlighting and developing outsider voices. A tendency that aided her biotech startup team at VLST (her title: Head of Ops) grow to 50 employees and $35M in funding.\n\nObsessed with growth and knowledge, Elizabeth went from a degree in Biochemistry, to Georgetown’s Global Executive MBA in Innovation, Entrepreneurship, Emerging Markets. She was most recently invited to join the 2019 cohort of Harvard Business School’s YALP program which aims to develop leaders who understand cross-sector collaborations for shared prosperity and can implement them more effectively and spread them more rapidly than in the past.\n\nDespite an active travel schedule overseeing labs and showcasing Seattle talent in SF, LA, NYC and internationally, Elizabeth is a hometown hero. She grew up in Seattle, plays the connector and mentor role in building Seattle’s booming economy, and she lives on Beacon Hill with her wife and toddler. Sundays usually mean farmer’s markets and family brunch."}]'
```

Comment:
This isn't the most elegant solution, but the way we handle the `title` field is now very specific to work right. Board Members must follow this pattern:
```
"title": ["Board Title", "Professional Title"]
```
If either field isn't defined - aka they don't have a specific board or professional title listed - it must be replaced by an empty string. For example,
```
title: ["Board Title", ""]
title: ["", "Professional Title"]
```